### PR TITLE
Add mdharamdas to pullapprove

### PR DESCRIPTION
**Pull request by gigawhitlocks**

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ groups:
       - mhagen-vmware
       - rajanashok
       - rogeliosanchez
+      - mdharamadas
```
Comment (sgairo) [inline .pullapprove.yml]: mdharamadas1

Comment (sgairo) [inline .pullapprove.yml]: I accidentally gave you his vmw not github

**Comment by gigawhitlocks** [comment]
lol forgot no ci qq

**Comment by anchal-agrawal** [comment]
Can we address this too with this PR? https://github.com/vmware/vic/pull/5852

**Pull request merged**